lnwallet/interface_test.go

```diff
@@ -1233,6 +1238,141 @@ func testSignOutputPrivateTweak(r *rpctest.Harness, w *lnwallet.LightningWallet,
 	}
 }
 
+func testReorgWalletBalance(r *rpctest.Harness, w *lnwallet.LightningWallet, t *testing.T) {
+	t.Logf("Running reorg wallet balance test")
+
+	// We first mine a few blocks to ensure any transactions still in the
+	// mempool confirm, and then get the original balance, before a
+	// reorganization that doesn't invalidate any existing transactions or
+	// create any new non-coinbase transactions. We'll then check if it's
+	// the same after the empty reorg.
+	_, err := r.Node.Generate(5)
+	if err != nil {
+		t.Fatalf("unable to generate blocks on passed node: %v", err)
+	}
+	// Give wallet time to catch up.
+	time.Sleep(time.Second)
+	// Send some money from the miner to the wallet
```

Comment:
Minor nit: missing a new line above here, and also below the call to `Generate`. There're a few other instances of this within this PR. However, to avoid notification spam, I'll only comment once.